modules/dashboard/test/TestPlan.md

```diff
@@ -1,56 +1,4 @@
 1. Log in. Note the time. Log out and log back in after 2 minutes. Check that welcome panel info is correct.
```

Comment:
@HenriRabalais the deletions seem to make sens as they all relate to widgets now included in other modules.

However, I'm wondering if we should add a line item to test the "loading a widget" functionality... technically it's already being tested by the modules themselves...

@driusan thoughts ?

Reply:
It think it would be a good idea to add tests for loading widget types/permissions. You can look at the `candidate_profile` test as an example.

Reply:
(I don't mind if that's not done in this PR since, as you say, the individual widget tests in the other PRs are implicitly testing that right now..)